koncept/inkludera-kapitel.tex

```diff
@@ -16,11 +16,8 @@
 % Avsnitt 1.8 Modulation
 \input{koncept/chapter1-8}
 % Avsnitt 1.9 Effekt och energi
-% Avsnitt 1.10 dB med miniräknare
-% Avsnitt 1.11 Decibel över 1 mW vid 50 ohm
-% Avsnitt 1.12 Sambandet spänning och dBm
 \input{koncept/chapter1-9}
-% Avsnitt 1.13 DSP
+% Avsnitt 1.10 DSP
 \input{koncept/chapter1-10}
```

Comment:
**Suggestion:** It seems like you have removed some comments that were indicating the content of the included chapters. If these comments were useful for navigation or understanding the document structure, consider updating them to reflect the current state of the document instead of removing them. [maintainability]
```suggestion
% Avsnitt 1.10 dB med miniräknare
% Avsnitt 1.11 Decibel över 1 mW vid 50 ohm
% Avsnitt 1.12 Sambandet spänning och dBm
% Avsnitt 1.13 DSP
\input{koncept/chapter1-10}
```